manifests/server/etcd/setup.pp

```diff
@@ -34,6 +69,13 @@
   Optional[Enum['existing', 'new']] $initial_cluster_state = undef,
   Optional[String[1]] $initial_cluster_token               = undef,
   Array[String[1]] $initial_cluster                        = [],
+
+  Optional[Stdlib::Unixpath] $binary_path = undef,
+  Stdlib::Unixpath $storage_path          = '/var/lib/etcd',
+  String[1] $user                         = $k8s::server::etcd::user,
+  String[1] $group                        = $k8s::server::etcd::group,
+  Integer[0, 65535] $uid                  = 113,
+  Integer[0, 65535] $gid                  = 121,
```

Comment:
where are those ids coming from? Do we need to set them? I want to ensure that we don't have collisions with other software. On Arch Linux for example, 113 is a reserved uid for unifi and 121 a reserved gid for lxdm.

Reply:
i took them from the ubuntu etcd package.